### PR TITLE
Admin api add bike

### PR DESCRIPTION
**Pull request by tmassalski**

### CODE REVIEW

File: src/app.module.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+
+import { BikesModule } from './bikes/bikes.module';
 
 @Module({
-  imports: [],
-  controllers: [AppController],
-  providers: [AppService],
+  imports: [
+    BikesModule,
+    MongooseModule.forRoot('mongodb+srv://test:test@cluster0.tnyju.mongodb.net/bikely?retryWrites=true&w=majority'),
```
Comment (MarKust71) [inline src/app.module.ts:9]: it's dangerous to publish such things but I know what is going to happen soon ;).

File: src/utils/test-utils.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import * as request from 'supertest';
+import { INestApplication, ModuleMetadata } from '@nestjs/common/interfaces';
+import { Test } from '@nestjs/testing';
+import { MongooseModule } from '@nestjs/mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+
+export async function testModuleWithInMemoryDb(moduleMetadata: ModuleMetadata) {
+  const mongoServer = new MongoMemoryServer();
+  const uri = await mongoServer.getConnectionString();
```
Comment (MarKust71) [inline src/utils/test-utils.ts]: Tools in my editor complain it's deprecated. Try to use `grtUri(...` instead.

**Review by MarKust71 — DISMISSED** [review]

**Review by MarKust71 — CHANGES_REQUESTED** [review]
better this time...

File: src/utils/test-utils.ts (hunk 2 above)
Comment (tmassalski) [reply src/utils/test-utils.ts]: Fixed.  Secon deprecation warning should be fixed with the next mongoose release.

File: src/app.module.ts (hunk 1 above)
Comment (tmassalski) [reply src/app.module.ts:9]: Right, This is only test DB. Will be removed with next PR.

**Review by MarKust71 — APPROVED** [review]

**Pull request merged**